src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.test.jsx

```diff
@@ -270,4 +271,79 @@ describe('<CustomFieldInputMultipleChoice>', () => {
       expect(screen.getByText('Choice 2')).toBeInTheDocument();
     });
   });
+
+  describe('dropdown close behavior', () => {
+    it('closes the dropdown when clicking outside', async () => {
+      const choices = [{
```

Comment:
2cents: The `requireProps` already has a list of choices. We don't necessarily need to define new ones. It makes the test a little less daunting.